### DISCUSSION TITLE
Starred repos don't show in Stars page.

### DISCUSSION THREAD

**Discussion by boing102** [community/community#39727] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hey,

I've noticed something strange on my Stars page today. I have the following repo starred:
<img width="2545" alt="Screenshot 2022-11-22 at 08 28 26" src="https://user-images.githubusercontent.com/6180524/203251686-841a2801-af70-41fd-916f-17350052a1b0.png">

But it actually doesn't show up in my Star page on my profile...
<img width="939" alt="Screenshot 2022-11-22 at 08 29 13" src="https://user-images.githubusercontent.com/6180524/203251834-60b2b22c-0f1a-4902-8cba-3ffe7eead1d7.png">

Am i doing something wrong or is this a bug?

**Comment by radar-blip** [comment]
I dont see it anywhere in your repositories, maybe thats why it isn't showing up.

**Reply by boing102** [reply]
It's not my repo, but a repo of an organization I'm part of.

**Reply by boing102** [reply]
And I actually noticed a different problem. In the screen below notice the `zdi` repo. 
<img width="929" alt="Screenshot 2022-11-23 at 11 53 45" src="https://user-images.githubusercontent.com/6180524/203529185-076ef7fc-1f7e-4827-ab6a-367f28d7e6bf.png">

But when I search for it, it doesn't show up 😅 
<img width="934" alt="Screenshot 2022-11-23 at 11 54 48" src="https://user-images.githubusercontent.com/6180524/203529451-af4aaecc-a90c-433e-afc2-5e21e03cd560.png">

**Reply by Bisjsidbdbdkdb** [reply]
Hii
